changelog/3356.bugfix.rst

```diff
@@ -0,0 +1,3 @@
+Fix off by one error in `sunpy.map.make_fiteswcs_header` where when using the
```

Comment:
`sunpy.map.make_fiteswcs_header` should be `sunpy.map.make_fitswcs_header`

Reply:
```suggestion
Fix off by one error in `sunpy.map.make_fitswcs_header` where when using the
```